src/ssmDocument/commands/createDocumentFromTemplate.ts

```diff
@@ -0,0 +1,119 @@
+/*!
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+import * as vscode from 'vscode'
+import * as YAML from 'yaml'
+import { getLogger, Logger } from '../../shared/logger'
+import * as picker from '../../shared/ui/picker'
+import { openAndSaveDocument } from '../util/util'
+
+import { getDocumentTemplate } from 'aws-ssm-document-language-service'
+
+export interface SsmDocumentTemplateQuickPickItem {
+    label: string
+    description: string
+    filename: string
+    language: string
+    docType: string
+}
+
+const SSMDOCUMENT_TEMPLATES: SsmDocumentTemplateQuickPickItem[] = [
+    {
+        label: localize('AWS.ssmDocument.template.command22Json.label', 'JSON Command Document 2.2'),
+        description: localize(
+            'AWS.ssmDocument.template.command22Json.description',
+            'An Example of a command document using schemaVersion 2.2 in JSON'
```

Comment:
From a UX perspective, if a user chooses to create a new document, they would then produce a document of a certain type. "An example of a" seems unnecessary.

Suggestion: "Command document (JSON, schemaVersion 2.2)", "Automation document (YAML, schemaVersion 0.3)"

Reply:
Another thing to consider in presenting the list is which axis should take precedence. Currently, the order appears to be "filetype, then purpose". If I'm thinking I want a command document, this could be messy to look through.

What are your thoughts about these ideas:
* "purpose then filetype" ordering
* splitting the "create" command into "create ssm doc (YAML)" and "create ssm doc (JSON)" commands

Reply:
I like you first suggestion, doing the "Command document (JSON, schemaVersion 2.2)", "Automation document (YAML, schemaVersion 0.3)"